README.md

```diff
@@ -2,7 +2,7 @@
 
 # <img alt="PhantomBot" src="https://i.zelakto.tv/7ZSJO3D.png" />
 
- [ ![Build Status](https://travis-ci.org/PhantomBot/PhantomBot.svg?branch=master)](https://travis-ci.org/PhantomBot/PhantomBot) 
+ [ ![Build Status](https://travis-ci.org/Phantombot/PhantomBot.svg?branch=master)](https://travis-ci.org/Phantombot/PhantomBot) 
```

Comment:
My mistake. I had used my own Travis before. Will fix

On Aug 31, 2016 08:23, "ScaniaTV" <notifications@github.com> wrote:

> In README.md
> <https://github.com/PhantomBot/PhantomBot/pull/833#discussion_r76930194>:
>
> > @@ -2,7 +2,7 @@
> >
> >  # <img alt="PhantomBot" src="https://i.zelakto.tv/7ZSJO3D.png" />
> >
> > - [ ![Build Status](https://travis-ci.org/PhantomBot/PhantomBot.svg?branch=master)](https://travis-ci.org/PhantomBot/PhantomBot)
> > + [ ![Build Status](https://travis-ci.org/Phantombot/PhantomBot.svg?branch=master)](https://travis-ci.org/Phantombot/PhantomBot)
>
> This needs to be PhantomBot. Phantombot breaks it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PhantomBot/PhantomBot/pull/833/files/e549df2094e20e8fb2123678b8c7395394db4ea7#r76930194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKxT7L5MLXhmx17zKTCUikA7wWlb0tEdks5qlR25gaJpZM4JxRTE>
> .
>